spending.py

```diff
@@ -0,0 +1,12 @@
+class Spending:
+    category: str
+    amount: float
+    date: str
+
+    def __init__(self, category: str, amount: float, date: str):
+        self.category = category
+        self.amount = amount
+        self.date = date
+
+    def to_str(self) -> str:
```

Comment:
class has __str__ dunder method which defines how class instance will be printed.

```
...
def __str__(self):
   return f"{self.category},{self.date},{self.amount}"
...


test = SomeClass()
print(test)
```